### PR TITLE
Fix changelog.txt path in generate_appdata_changelog

### PR DESCRIPTION
**Pull request by tux93**

### CODE REVIEW

**Comment by ilya-fedin** [comment]
Replace `..` with `${submodules_loc}`

**Comment by ilya-fedin** [comment]
Replace `..` with `${CMAKE_SOURCE_DIR}`

File: Telegram/CMakeLists.txt [hunk 1]
```diff
@@ -1320,7 +1320,7 @@ endif()
 if (LINUX AND DESKTOP_APP_USE_PACKAGED)
     include(GNUInstallDirs)
     configure_file("../lib/xdg/telegramdesktop.appdata.xml.in" "${CMAKE_CURRENT_BINARY_DIR}/telegramdesktop.appdata.xml" @ONLY)
-    generate_appdata_changelog(Telegram "../changelog.txt" "${CMAKE_CURRENT_BINARY_DIR}/telegramdesktop.appdata.xml")
+    generate_appdata_changelog(Telegram "../../changelog.txt" "${CMAKE_CURRENT_BINARY_DIR}/telegramdesktop.appdata.xml")
```
Comment (ilya-fedin) [inline Telegram/CMakeLists.txt:1323]:
```suggestion
    generate_appdata_changelog(Telegram "${CMAKE_SOURCE_DIR}/changelog.txt" "${CMAKE_CURRENT_BINARY_DIR}/telegramdesktop.appdata.xml")
```

Comment (tux93) [reply Telegram/CMakeLists.txt]: Thanks for the suggestion, applied

**Pull request merged**